### PR TITLE
Polish locales

### PR DESCRIPTION
**Pull request by kkarpieszuk**
hi there! I love your app! I created json file with Polish translations. It is translated from English but I truncated it tohas the same strings as Spanish has (I see it has no *_desc things, I believe Polish don't need it as well)

### CODE REVIEW

File: locales/pl.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+	"select-user-type.who_you_are": "Kim jesteś? (będziesz to mógł zmienić później)",
```
Comment (jytug) [inline locales/pl.json]: I'd suggest "będziesz mógł to zmienić później"

File: locales/pl.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	"select-user-type.driver": "Kierowca",
+	"select-user-type.passenger": "Pasażer",
+	"request-phone.type_your_phone": "Proszę przesłać do nas swój numer telefonu. Nie udostępnimy go bez twojej wyraźnej zgody.",
+	"request-phone.button_text": "Wyślij numer",
+	"request-phone.all_set": "👌 Wszystko jest już ustawione, możesz już zamówić przejazd!",
```
Comment (jytug) [inline locales/pl.json]: The "już" repetition is a stylistic error. My suggestion would be "Wszystko jest już ustawione, możesz zamówić przejazd!"

File: locales/pl.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	"passenger-index.choose_taxi_type": "Jaki chcesz rodzaj taksówki?",
+	"passenger-index.car": "🚗 Samochód",
+	"passenger-index.motorbike": "🏍 Skuter / Motocykl",
+	"passenger-index.s": "🔧 Ustawienia",
+	"passenger-index.error_only_known_type": "Nieznany typ, proszę wybrać samochód, motor lub ustawienia.",
+	"passenger-request-location.provide_location": "Aby znaleźć kierowców w pobliżu, proszę udostępnić swoją lokalizację.",
+	"passenger-request-destination.provide_destination": "Gdzie chcesz jechać? Proszę wpisać dokładną lokalizację (w sposób zrozumiały dla wyszukiwarki GPS) lub opisać to miejsce.",
```
Comment (jytug) [inline locales/pl.json]: "Dokąd"? :-)

File: locales/pl.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+	"driver-select-vehicle-type.motorbike": "🏍 Skuter / Motocykl",
+	"driver-select-vehicle-type.car": "🚗 Samochód",
+	"driver-select-vehicle-type.error_only_known_type": "Nieznany typ, proszę wybrać samochód lub motor.",
+	"driver-explain-checkins.text": "Prawie skończyliśmy, oto kilka spraw do odnotowania. Telegram nie sprawdza pozycji GPS na bieżąco. Zatem zamelduj się za każdym razem gdy zmienisz swoje miejsce postoju, a gdy jesteś w trybie oczekiwania na klienta, potwierdź swoją lokalizację co godzinę. To nam pomoże znaleźć pasujących pasażerów. Kliknij \"Dalej\".",
+	"driver-explain-checkins.next": "Dalej",
+	"driver-request-location.provide_location": "Aby pasażerowie mogli cię znaleźć, proszę udostępnić swoją aktualną lokalizację.",
+	"driver-explain-whats-next.text": "Wszystko ustawione! Powiadomimy cię o pasażerach lub zamówieniach w pobliżu. Nie wyłączaj tej aplikacji i zamelduj się co godzinę. Kliknij \"Dalej\".",
+	"driver-explain-whats-next.next": "Dalej",
+	"driver-index.checkin": "📍 Zamelduj się",
+	"driver-index.mute": "🔕 Uśpij",
```
Comment (jytug) [inline locales/pl.json]: I'd suggest "wycisz" for "mute" and "wyłącz wyciszenie" for unmute. This way you can rephrase the "driver-unmute.unmute_ok" string, so it sounds less awkward

File: locales/pl.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+	"driver-index.unmute": "🔔 Wybudź (wskazane)",
+	"driver-index.settings": "🔧 Ustawienia",
+	"driver-index.unknown_choice": "Nieznana opcja, spróbuj ponownie.",
+	"driver-mute.mute_ok": "⚠️ Pamiętaj jednak, że NIE otrzymasz powiadomień o nowych pasażerach dopóki nie wybudzisz aplikacji.",
+	"driver-unmute.unmute_ok": "Jesteś teraz wybudzony i będziesz otrzymywać powiadomienia o pasażerach w okolicy. Nie zapomnij meldować się co godzinę.",
```
Comment (jytug) [inline locales/pl.json]: If you choose to apply my last suggestion, this could go: "Wyłączyłeś wyciszenie i będziesz..."

File: locales/pl.json [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+	"driver-checkin.provide_location": "Udostępnij proszę swoją obecną pozycję tak by pasażerowie mogli cię znaleźć.",
+	"driver-checkin.ok": "👌 Dziękujemy za zameldowanie się. Zamelduj się co godzinę lub  za każdym razem gdy zmienisz lokalizację! Sprawdź także lokalizację powyżej, jeśli jest nieprawidłowa, zamelduj się ponownie lub kliknij spinacz do papieru (zamiast wcisnąć przycisk).",
+	"driver-order-new.new_order": "🔔 Nowe zamówienie!",
+	"driver-order-new.distance": "Odległość do pasażera: ~%s. Lokalizacja pasażera (kliknij aby przybliżyć):",
+	"driver-order-new.from": "Pasażer jest tutaj: %s",
+	"driver-order-new.to": "Pasażer jedzie do: %s",
+	"driver-order-new.price": "Pasażer chce zapłacić: %s (w lokalnej walucie)",
+	"driver-order-new.call_to_action": "Proszę wybrać działanie 👇 lub zignorować te powiadomienie",
```
Comment (jytug) [inline locales/pl.json]: "to powiadomienie"

File: locales/pl.json [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+	"driver-order-new.send_my_number": "Wyślij mój numer",
+	"driver-order-new.set_my_price": "Podaj własną cenę",
+	"driver-order-new.price_not_set": "Cena za ten przejazd nie została ustawiona przez pasażera, musisz podać własną cenę.",
+	"metric-distance.km": "%s km",
+	"passenger-request-price.set_price": "W tej aplikacji to pasażer podaje swoją cenę 😉 Podaj cenę (w lokalnej walucie). Ile chcesz zapłacić (lub wpisz \"0\" (zero) jeśli nie wiesz)",
+	"passenger-request-price.should_be_numeric": "Ups, cena musi być podana cyframi (i w lokalnej walucie)! Na przykład : \"50\". Proszę spróbować ponownie!",
+	"passenger-request-price.on_timeout": "Czekasz za długo? Przepraszamy. Jeśli nie znalazłeś przewozu, spróbuj ponownie później.",
+	"passenger-contact-new-number.message": "🔔 {{driver}} zaakceptował twoją cenę! Odległość do kierowcy: ~{{distance}}. Zadzwoń do kierowcy by potwierdzić przejazd:",
```
Comment (jytug) [inline locales/pl.json]: "Zadzwoń do kierowcy, by..."

File: locales/pl.json [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+	"driver-order-set-price.set_price": "Podaj własną cenę (w walucie lokalnej). Możesz podać swoje wytłumaczenie, na przykład: \"20, ponieważ most jest w remoncie\" lub \"15, jeśli nie będzie korków\"",
+	"passenger-contact-driver-price.message": "🔔 ⚠️ {{driver}} oferuje inną cenę: \"{{price}}\". Odległość do kierowcy: ~{{distance}}. Jeśli się zgadzasz, zadzwoń do kierowcy by potwierdzić przejazd:",
```
Comment (jytug) [inline locales/pl.json]: "zadzwoń do kierowcy, by..."

File: locales/pl.json [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+	"passenger-verify-location.verify_location": "Spójrz na mapę powyżej (kliknij aby przybliżyć). Czy ta lokalizacja jest prawidłowa?",
```
Comment (jytug) [inline locales/pl.json]: "kliknij, aby"

File: locales/pl.json [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+	"passenger-verify-location.try_with_paperclip": "Spróbuj wysłać swoją lokalizację przez kliknięcie w spinasz do papieru, a następnie ikonie lokalizacji.",
```
Comment (jytug) [inline locales/pl.json]: spinacz

File: locales/pl.json [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+	"passenger-verify-cash.verify_cash": "Większość kierowców nie akcpetuje kart kredytowych. Czy masz ze sobą gotówkę?",
```
Comment (jytug) [inline locales/pl.json]: akceptuje

File: locales/pl.json [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+	"passenger-verify-cash.get_some_cash": "🙁 Przykro nam, musisz mieć ze sobą gotówkę. Spróbuj wypłacić trochę pieniędzy i dopiero wtedy poszukać kierowcy!",
+	"driver-request-radius.request_radius": "Jak daleko możesz podjechać po pasażera? Wybierz z listy 👇 lub wpisz liczbę (do 10 km).",
+	"driver-request-radius.option1": "1 km (~0.6 mili)",
+	"driver-request-radius.option3": "3 km (~1.8 mili), rekomendowane",
```
Comment (jytug) [inline locales/pl.json]: Stick to one translation of "recommended". I think "wskazane" sounds nice

File: locales/pl.json [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+	"driver-request-radius.option5": "5 km (~3.1 mili)",
+	"driver-request-radius.should_be_valid": "Nie rozumiem tego wyboru. Proszę wybrać  opcję  👇 lub wpisać liczbę od 1 do 10.",
+	"global.location_button_text": "Udostępnij położenie",
+	"global.default_options_message": "Proszę wybrać działanie z menu poniżej 👇",
+	"global.default_inline_options_message": "Kliknij przycisk 👇 lub zignoruj ten komunikat",
+	"global.replied_to_order": "✔️ Odpowiedziałeś na te ☝️ zamówienie. Oczekuj na telefon od pasażera w celu potwierdzenia.",
```
Comment (jytug) [inline locales/pl.json]: "to zamówienie"

File: locales/pl.json [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+	"global.yes": "Tak",
+	"global.no": "Nie",
+	"global.error_try_again": "Ups, nie rozumiem tego. Spróbuj ponownie.",
+	"global.error_location": "Ups, nie rozpoznaję tego położenia. Kliknij przycisk lub użyj ikony spinacza do papieru by udostępnić koordynaty GPS.",
```
Comment (jytug) [inline locales/pl.json]: "użyj ikony spinacza, by udostępnić **współrzędne**

**Comment by kkarpieszuk** [comment]
thanks for review! I committed new changes according to your suggestions

**Comment by kkarpieszuk** [comment]
I am not sure if you get notification about this so I will ping you @jytug  ;)

**Comment by kkarpieszuk** [comment]
hi, will you merge it? :)

**Comment by ro31337** [comment]
Thanks for contribution, I'll merge it this weekend. Took a while because
of some Telegram limitations (no paging for language buttons). Will make a
room for 5 more languages and merge. Meanwhile, you may want to take a look
at https://github.com/ro31337/libretaxi-www . I appreciate if you can
translate index.html. Thank you!

**Comment by ro31337** [comment]
Thanks for contribution, file was merged. However, will be integrated here: https://github.com/ro31337/libretaxi/issues/517

**Pull request merged**